Jellyfin.Api/Controllers/Images/ImageByNameController.cs

```diff
@@ -0,0 +1,226 @@
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Mime;
+using MediaBrowser.Controller;
+using MediaBrowser.Controller.Configuration;
+using MediaBrowser.Controller.Entities;
+using MediaBrowser.Model.Dto;
+using MediaBrowser.Model.IO;
+using MediaBrowser.Model.Net;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Jellyfin.Api.Controllers.Images
+{
+    /// <summary>
+    ///     Images By Name Controller.
+    /// </summary>
+    [Route("Images")]
+    [Authorize]
```

Comment:
am I right that this requires the requester to be logged in? it's a change in behaviour (though I _think_ we can leave it as so)

Reply:
You're correct, and I do think we can make it so all of these endpoints require auth. I found it odd that a few endpoints didn't require it